### ISSUE TITLE
Request for more customization

### ISSUE THREAD

**Issue by hero887** [Geforce132/SecurityCraft#15]
I am wanting to use SecurityCraft as a player managed anti-grief mod for my server. The gist of my idea is to disable all the crafting recipes for reinforced items and require the player to get the Universal Reinforcer LvL1 to upgrade their bases into grief free bases. I am using MineTweaker3 to change the recipes. It seems I have been able to remove all the reinforced items but I haven't been able to remove the Universal Reinforcer LvL1 recipe. What I want to do with that item is make it more expensive for players to make. I like the heart take-a-way feature when using it tho, it's more balanced that way. Good thinking on that!

Long story short, MineTweaker3 can't seem to change it's recipe as it does with other items in MC. I suppose that's due to mod support issues. So that's why I'm requesting for a more configureable settings to remove certain items or change recipes if desired.

What do you think? Can this be done? I'm using SecurityCraft v1.8.0.

**Comment by bl4ckscor3** [comment]
Configurable recipes and disabling items is not something that's easily manageable in a user friendly fashion.
